script/create-dist.py

```diff
@@ -11,8 +11,10 @@
 if sys.platform == "win32":
   import _winreg
 
-from lib.config import BASE_URL, PLATFORM, enable_verbose_mode, \
-                       get_target_arch, get_zip_name, build_env
+from lib.config import BASE_URL, PLATFORM, add_exec_bit, \
```

Comment:
`add_exec_bit` is defined in lib.util, not lib.config.

Reply:
Good catch! Fixed.